manifests/monitoring/grafana/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
         - name: grafana
-          image: "grafana/grafana:7.2.1"
+          image: "grafana/grafana:7.5.5"
```

Comment:
This is not OK, we can't use Grafana under AGPL, the last Grafana version released under Apache is v7.5.4.